test/inductor/test_torchinductor.py

```diff
@@ -2770,8 +2770,6 @@ def fn(x, y):
         if self.device != "cpu":
             self.assertEqual(torch._inductor.metrics.generated_kernel_count, 1)
 
-    # FIXME: https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/3378
-    @skipCUDAIfRocm
```

Comment:
So the few tests you aren't skipping pass now?

Reply:
Correct, these UTs pass now:
```
test_gather_scatter - PASSED
test_index_put2  - PASSED
test_argmax_argmin1 - PASSED
test_argmax_argmin2 - PASSED
```